File: python/credit_check.py
def credit_check(string):
    doubled_num = []
    added_doubles = []
    str_list = list(string)
    final_check = 0
    for i in range(0, len(str_list), 1):
        if i % 2 == 0:
            doubled_num.append(str(int(str_list[i])*2))
        else: doubled_num.append(str(str_list[i]))
    for j in range(0, len(doubled_num), 1):
        if len(doubled_num[j]) == 2:
            added_doubles.append((int(doubled_num[j][0])) + (int(doubled_num[j][1])))
        else: added_doubles.append(int(doubled_num[j]))
    for k in added_doubles:
        final_check += k
    if final_check % 10 == 0:
        return "The number is valid!"
    else: return "The number is invalid!"

File: js/creditCheck.js
exports.creditCheck = function(str) {
    let doubledNum = [], addedDoubles = []
    let strArr = str.split('')
    let finalCheck = 0
    for(let i = 0; i < strArr.length; i++){
        if(i % 2 === 0){
            doubledNum.push((strArr[i]*2).toString())
        }else doubledNum.push(strArr[i])
    }for(let j = 0; j < doubledNum.length; j++){
        if(doubledNum[j].length === 2){
            addedDoubles.push(+doubledNum[j][0] + +doubledNum[j][1])
        }else{addedDoubles.push(+doubledNum[j])}
    }for(let k of addedDoubles){
        finalCheck += k
    }
   if(finalCheck % 10 === 0){
       return "The number is valid!"
   }
   else return "The number is invalid!"
}